mods/ra/rules/infantry.yaml

```diff
@@ -388,6 +388,8 @@ DELPHI:
 
 CHAN:
 	Inherits: ^CivInfantry
+	Selectable:
+		Class: CHAN
```

Comment:
From my understanding you don't need to define `Class` if you want to use the actor definition's name, which is what this is doing.